Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+const { AuthMethod, localDevUri } = require("../../../support/constants");
+
+const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
+
+const dobYearSelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectYear]";
+const dobMonthSelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectMonth]";
+const dobDaySelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectDay]";
+const dovYearSelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectYear]";
+const dovMonthSelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectMonth]";
+const dovDaySelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectDay]";
+
+const feedbackPhnMustBeValidSelector = "[data-testid=feedbackPhnMustBeValid]";
+const feedbackPhnIsRequiredSelector = "[data-testid=feedbackPhnIsRequired]";
+const feedbackDobIsRequiredSelector = "[data-testid=feedbackDobIsRequired]";
+const feedbackDovIsRequiredSelector = "[data-testid=feedbackDovIsRequired]";
+
+const homeUrl = "/";
+const vaccineCardUrl = "/vaccinecard";
+
+const vaccinationStatusModule = "VaccinationStatus";
+
+function enterVaccineCardPHN(phn) {
+    cy.get("[data-testid=phnInput]")
+        .should("be.visible", "be.enabled")
+        .type(phn);
+}
+
+function select(selector, value) {
+    cy.get(selector).should("be.visible", "be.enabled").select(value);
+}
+
+function selectExist(selector, value) {
+    cy.get(selector)
+        .children("[value=" + value + "]")
+        .should("exist");
+}
+
+function selectNotExist(selector, value) {
+    cy.get(selector)
+        .children("[value=" + value + "]")
+        .should("not.exist");
+}
+
+function clickVaccineCardEnterButton() {
+    cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible").click();
+}
+
+describe("Vaccine Card Page", () => {
+    it("Landing Page - Vaccine Card Button does not exist - Vaccine Status module disabled", () => {
+        cy.enableModules({});
+        cy.visit(homeUrl);
+        cy.get("[data-testid=btnVaccineCard]").should("not.exist");
+    });
+
+    it("Landing Page - Log In - Vaccine Status module disabled and vaccine card URL entered directly", () => {
+        cy.enableModules({});
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=BCSCBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=IDIRBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=KeyCloakBtn]").should("be.visible", "be.enabled");
```

Comment:
This test is relying on elements that aren't directly related to what it should be testing, which makes it fragile.  Also, the login page only has these buttons in DEV.  I'd suggest swapping these out and just checking that the "[data-testid=vaccineCardFormTitle]" element does not exist.

Reply:
Changed.